.github/workflows/ci.yml

```diff
@@ -127,21 +127,21 @@ jobs:
 
     # Install arm64 cross-compilation toolchain if required
     - name: Install arm64 cross-compilation toolchain
-      if: runner.os == 'Linux' && matrix.arch == 'arm64'
+      if: runner.os == 'Linux' && matrix.os == 'ubuntu-20.04'
```

Comment:
This  if condition should be kept as it was, we don't need to install these on the x64 build.

Also, why the change to the packages installed below? It looks like `g++-aarch64-linux-gnu` depends on `gcc-aarch64-linux-gnu` so we shouldn't need to depend on that explicitly. Given the build is working, maybe we don't need `pkg-config-aarch64-linux-gnu`, but I'd be more comfortable leaving that is.